Firestore/core/src/util/ordered_code.cc

```diff
@@ -16,20 +16,26 @@
 
 #include "Firestore/core/src/util/ordered_code.h"
 
+#include <limits>
+
```

Comment:
Is there a reason to skip `#include <cstdint>` Although it might be included in some other libraries but according to what I found online: `that is not something you can rely on. Standard library implementors may one day decide that they will no longer do it like that.`

Reply:
It's because the internal version used c style, i changed it back. Seems to be more sensesible.